python/action_server/config.py

```diff
@@ -0,0 +1,66 @@
+from pydantic import BaseModel
+import zenoh
+import json
+from typing import Tuple
+
+zenoh.init_logger()
+
+class ZenohValidator(BaseModel):
+    '''
+    Config module for zenoh configuration. Validates the configuration variables using ConfZ by pydantic module.
+    Args:
+        mode(str): two choices were there:
+            - peer
+            - client
+        connect(str): the endpoint to connect
+        listen(str): the endpoint to listen 
+        config(str): a configuration file for zenoh variables.
+        base_key_expr(str): a common keyexpression to use. 
+    '''
+    mode: str
+    connect: str
+    listen: str
+    base_key_expr: str
+    complete: bool
+
+class ZenohConfig:
+    '''
+    Class creates zenoh configuration object and validates tha configuration variables.
+    '''
+    def __init__(self,
+                mode: str = "peer",
+                connect: str = "",
+                listen: str = "",
+                config: str = "",
+                base_key_expr: str = "Genotyper/1/DNASensor/1",
+                complete: bool = False) -> None:
```

Comment:
what is this and why is it needed?

Reply:
Complete is a parameter in queryable to declare the complete storage to that queryable. But it is not needed now so, I have removed it.